demo/plugins.txt

```diff
@@ -0,0 +1,41 @@
+ace-editor:1.0.1
+async-http-client:1.7.8
+authentication-tokens:1.2
+branch-api:0.2-beta-7
```

Comment:
:bug: too old.

In general, after running the demo, look for all available updates of plugins listed in this file, and bump up the revisions in the image.